CHANGES

```diff
@@ -75,6 +75,8 @@ Features added
 * #4183: doctest: ``:pyversion:`` option also follows PEP-440 specification
 * #4235: html: Add :confval:`manpages_url` to make manpage roles to hyperlinks
 * #3570: autodoc: Do not display 'typing.' module for type hints
+* #4271: sphinx-build supports an option called ``-j0`` to adjust numbers of
+  processes automatically.
```

Comment:
This says `-j0`, yet the code says `-jauto`. Which one is correct? Maybe both?

Reply:
It's my wrong. `-j auto` is still correct. But we need to choose which is better.

Note: I think supporting both is worse. we should choose one. I love python's slogan; [TOOWTDI](https://wiki.python.org/moin/TOOWTDI)

Reply:
Ah, personally I'd consider supporting both a usability improvement (`-j0` is more concise and can be used without the space, but `-j auto` is clearer), however, I wouldn't block on it. We can extend it in the future if anyone wants it.